lib/opte-ioctl/src/lib.rs

```diff
@@ -396,7 +396,7 @@ where
 }
 
 pub fn fetch_fragile_types() -> Result<FragileInternals, Error> {
-    let dld_ctf = Ctf::from_file("/kernel/drv/amd64/dld")
+    let dld_ctf = Ctf::from_file("/system/object/dld/object")
```

Comment:
😮 TIL objfs

Reply:
I learned that trick from Robert! I think it'll be short-lived if we use your below suggestion. I'll preserve a copy of this branch since we could end up needing these tricks again (although I hope we do not).